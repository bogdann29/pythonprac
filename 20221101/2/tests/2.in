t1 = Triangle((0, 0), (0, 0), (0, 0))
t2 = Triangle((0, 6), (9, 1), (5, 2))
print(t1 in t2, abs(t1), abs(t2))
print(bool(t1), bool(t2))
print(t1 < t2, t2 < t1, t1 & t2)