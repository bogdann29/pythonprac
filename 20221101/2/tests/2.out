True 0 5.500000000000004
False True
True False False
